certified-connectors/Cascade Strategy/apiDefinition.swagger.json

```diff
@@ -411,19 +454,56 @@
                       }
                     }
                   },
-                  "description": "historical_data"
+                  "description": "Historical Data"
                 }
               },
               "required": [
                 "workspace_id",
                 "plan_id"
               ]
             }
+          },
+          {
+            "name": "measureId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Measure",
+            "x-ms-url-encoding": "single",
+            "description": "Select Measure to be updated after selecting Plan.",
+            "x-ms-dynamic-values": {
+              "operationId": "GetMeasuresByPlan",
```

Comment:
The path template has been changed for the operation. ValuePath: operation/GetMeasuresByPlan, can you please confirm that no existing flows or users will be affected by this change?